site/docs/patterns/contact-details.mdx

```diff
@@ -0,0 +1,131 @@
+---
+title: Contact Details
+layout: DetailPattern
+aliases:
+  - /salt/patterns/contact-details
+data:
+  resources:
+    [
+      { href: "https://storybook.saltdesignsystem.com/?path=/story/patterns-contact-details--contact-details", label: "Examples" },
+    ]
+---
+
+Contact Details is a pattern that displays communication information—for a person, company, distribution list or individual channel, such as a name, a phone number or email address.
+
+ <Diagram
+    src="/img/patterns/contact-details/contact-details.png"
+    alt="Contact details how to build"
+    border
+  />
+
+We provide design guidance for using Salt [Avatar](/salt/components/avatar), [Button](/salt/components/button), [Icon](/salt/components/icon), [Text](/salt/components/text), [Tooltip](/salt/components/tooltip) and Layout components to achieve the pattern. You can also find implementation examples in the [storybook](https://storybook.saltdesignsystem.com/?path=/story/patterns-contact-details--contact-details)
```

Comment:
We provide design guidance for using Salt [Avatar](https://github.com/salt/components/avatar), [Button](https://github.com/salt/components/button), [Stack Layout](https://github.com/salt/components/stack-layout), [Icon](https://github.com/salt/components/icon), [Text](https://github.com/salt/components/text), and [Tooltip](https://github.com/salt/components/tooltip) components to achieve the pattern. You can also find implementation examples in the [storybook](https://storybook.saltdesignsystem.com/?path=/story/patterns-contact-details--contact-details)
```suggestion
We provide design guidance for using Salt [Avatar](/salt/components/avatar), [Button](/salt/components/button), [Icon](/salt/components/icon), [Text](/salt/components/text), [Tooltip](/salt/components/tooltip) and Layout components to achieve the pattern. You can also find implementation examples in the [storybook](https://storybook.saltdesignsystem.com/?path=/story/patterns-contact-details--contact-details)
```

Reply:
I thought it made more sense to reference Layout components generally instead of specifically listing out Stack and Flow Layouts